src/data/bool.lean

```diff
@@ -132,6 +132,10 @@ theorem band_elim_right : ∀ {a b : bool}, a && b → b := dec_trivial
 
 @[simp] theorem bnot_true : bnot tt = ff := rfl
 
+theorem bnot_iff_not : ∀ {b : bool}, !b ↔ ¬b
```

Comment:
Is this worth considering:
```suggestion
@[simp] theorem not_iff_bnot : ∀ {b : bool}, ¬b ↔ !b
```
(I've never used `bool` so I don't really know what the design intentions in this part of the library are.)

Reply:
I think the original direction is better. If you're using a bool as a prop, make as much of it a prop as possible.

Reply:
OK sure! Thanks for providing guidance :)